pyscriptjs/tests/integration/test_py_config.py

```diff
@@ -0,0 +1,43 @@
+import tarfile
+from io import BytesIO
+
+import requests
+
+from .support import PyScriptTest
+
+URL = "https://github.com/pyodide/pyodide/releases/download/0.20.0/pyodide-build-0.20.0.tar.bz2"
+
+
+def download_and_unzip(url, extract_to="."):
+    response = requests.get(url, stream=True)
+    file = tarfile.open(fileobj=BytesIO(response.raw.read()), mode="r:bz2")
+    file.extractall(path=extract_to)
+
```

Comment:
good job! I like the fact that the test does the setup automatically without requiring the user intervention :+1: 

It's a bit unfortunate that you have to re-download the same big tarfile again and again in case you are running the tests locally and multiple times.
One quick&dirty solution is to save the tarball to e.g. `/tmp/pyscript-test-cache/pyodide...tar.bz2`: if it's already there, we just use it, else we proceed with the download.
It's a bit dirty because it assumes that you have write access to `/tmp/pyscript-test-cache`, but I guess this is true for all linux and macs. It might fail on windows but I guess it's fine not to care for now.
(A less dirty solution would be to use `import tempfile` to create a proper temporary directory and `pytest-cache` to remember its location between invocations, but I think that the quick&dirty solution is fine for now :))

Reply:
done, thanks!